signer/src/bitcoin/utxo.rs

```diff
@@ -982,18 +993,18 @@ impl BitcoinTxInfo {
     ///
     /// The logic for the fee assessment is from
     /// <https://github.com/stacks-network/sbtc/issues/182>.
-    pub fn assess_output_fee(&self, vout: usize) -> Option<Amount> {
+    fn assess_output_fee(&self, vout: usize, fee: Amount) -> Option<Amount> {
```

Comment:
suggestion: I'd rename the `fee` parameter to `tx_fee` or something so that it's clearer as to _which_ fee this is.

Same above with `assess_input_fee()`

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/803/commits/84f9fd7bc05073500cd61d46d0a567d77b4f6b18